client/faucet/src/mint.rs

```diff
@@ -7,24 +7,46 @@ use libra_types::account_config::{
     testnet_dd_account_address, treasury_compliance_account_address, type_tag_for_currency_code,
     LBR_NAME,
 };
-use std::convert::From;
+use std::{convert::From, fmt};
+
+#[derive(Debug)]
+pub enum Response {
+    DDAccountNextSeqNum(u64),
+    SubmittedTxns(Vec<libra_types::transaction::SignedTransaction>),
+}
+
+impl std::fmt::Display for Response {
```

Comment:
should we do impl FromStr for Response instead ?

Reply:
no, because I don't expect create Response from string

Reply:
sorry, what I meant is actually implement to_string() https://doc.rust-lang.org/std/string/trait.ToString.html , Display is used for using in an format string that uses "{}", which I don't think is what we need here.

Reply:
nvm, found out